mix.exs

```diff
@@ -46,7 +46,8 @@ defmodule Still.MixProject do
       extra_applications: [:logger],
       mod: {Still.Application, []},
       env: [
-        server: false
+        server: false,
+        profiler: false
```

Comment:
I guess we also want this on the generator's template?

Reply:
It already is.

Reply:
It is? I don't see the change

Reply:
It was in the original commit. You can find it under `installer/templates/config`

Reply:
Oh,  you already merged it